contributors/triage/README.md

```diff
@@ -0,0 +1,13 @@
+# Kubernetes Issue Triage
```

Comment:
@bgrant0607 that's a good point to put the content under contributors/devel. Jorge and I thought about it, but we also thought let's keep Triage on it's own directory in case we are adding more specific content in future and keep it separate from pure devel docs. The issue.md is mentioned in the guidelines as a reference to learn about priorities in detail but if you think it's better to modify issues.md as guidelines that's fine with me. If contributors/devel seems better and right place, I will move the guideline there.  And btw, I couldn't open the #card-2296048 link you have provided above. Thanks!